podcasts/en.lproj/Localizable.strings

```diff
@@ -3507,7 +3507,7 @@
 "eoy_story_listened_to_updated" = "This year, you spent %1$@ listening to podcasts";
 
 /* Text that appears when someone shares the listened to story to Twitter, for example. %1$@ is a placeholder for the amount of time. */
-"eoy_story_listened_to_share_text" = "I spent %1$@ listening to podcasts in 2022";
+"eoy_story_listened_to_share_text" = "I spent %1$@ listening to podcasts this year";
```

Comment:
@ashiagr I changed some of the copies when sharing. Let me know what do you think and happy to change anything if you'd like to!